flash/image/style_transfer/backbone.py

```diff
@@ -0,0 +1,24 @@
+from flash.core.registry import FlashRegistry
+from flash.core.utilities.imports import _PYSTICHE_AVAILABLE
+
+STYLE_TRANSFER_BACKBONES = FlashRegistry("backbones")
+
+__all__ = ["STYLE_TRANSFER_BACKBONES"]
+
+if _PYSTICHE_AVAILABLE:
+
+    from pystiche import enc
+
+    for mle_fn in dir(enc):
+
+        if "multi_layer_encoder" not in mle_fn:
+            continue
+
+        name = mle_fn.split("_")[0]
```

Comment:
This will result in the same name for `vgg19` and `vgg19_bn`. Why not keep the regex matching?

Reply:
Reverted back.